minio/minio
# fix success alert not shown after object is uploaded

## Description

After an object is uploaded successfully, a success alert should be shown.

## Motivation and Context


Before fetching the list of objects, we clear any outstanding alerts. This is done to remove obsolete alerts if there are any. But in case of upload, we show a success alert and then refresh object list. Here, success alert is cleared and user is not seeing any response to upload action. This is fixed by adding auto clear to the alert. The alert will be cleared automatically after 5 seconds.

## Regression


Yes #6385 

## How Has This Been Tested?




1. Run minio as s3 gateway
2. Create a bucket
3. Set Deny ListObjects policy on the bucket
4. Open minio browser and click on the bucket.
5. Error message shown and object list is empty
6. Error message is cleared after 5 seconds
7. Create a new bucket
8. Upload a file to that bucket
9. See success alert is shown after the file is uploaded

## Types of changes


## Checklist:
